Sources/Transaction/Fee/BlockchainMetaFetcher.swift

```diff
@@ -116,7 +111,13 @@ final class BlockchainMetaFetcher {
                 let responseFee = response.mobMinimumFee
                 let minimumFee = responseFee != 0 ? responseFee : McConstants.DEFAULT_MINIMUM_FEE
                 let blockVersion = BlockVersion(response.networkBlockVersion)
-                self.cacheMeta(minimumFee: minimumFee, blockVersion: blockVersion) {
+                let minimumFees = response.minimumFees.reduce(into: [TokenId: UInt64](), {
+                    $0[TokenId($1.key)] = $1.value
+                })
+                self.cacheMeta(
+                    minimumFee: minimumFee,
+                    minimumFees: minimumFees,
+                    blockVersion: blockVersion) {
                     completion(.success($0))
```

Comment:
We could drop minimumFee here and just use minimumFees everywhere else, if we just check for the presence of .MOB in the response and then set to the DEFAULT_MINIMUM_FEE if a value for .MOB is not present.

i.e. just set the value for .MOB in the fees if it doesn't exist...

Also, for the single minimum fee value, the logic will set to a minimum if the value is zero, but if we receive `minimumFees` in the response, we do not override a value of zero with DEFAULT_MINIMUM_FEE like we do for the single fee version of the code.   Should they be the same with respect to the default value logic?

Reply:
ok ya I like your idea, will simplify some of the code paths. Ill look into override going to zero, it should be going
 `DEFAULT_MINIMUM_FEE`